modules/solid_mechanics/doc/content/source/postprocessors/Mass.md

```diff
@@ -1 +1,13 @@
-!template load file=stubs/moose_object.md.template name=Mass syntax=/Postprocessors/Mass
\ No newline at end of file
+# Mass
+
+!syntax description /Postprocessors/Mass
+
+!alert note
+The `ADMass` postprocessor may be used with a density [AD](automatic_differentiation/index.md) material
```

Comment:
Is this postprocessor only in `AD`? Is automatic differentiation required here?

Reply:
Mass will work with non-AD density
ADMass will work with AD density
there's no doco page for ADmass